HousingManagementSystemApi/UseCases/RetrieveAddressesUseCase.cs

```diff
@@ -39,9 +42,16 @@ public async Task<IEnumerable<PropertyAddress>> Execute(string postcode)
                 var asset = await assetGateway.RetrieveAsset(property.Reference.ID);
                 if (assetTypes.Contains(asset.AssetType))
                 {
-                    filteredAssets.Add(property);
+                    var tenureInformation = await this.tenureGateway.RetrieveTenureType(asset.Id);
```

Comment:
The argument passed in should be `asset.Tenure.Id` to reflect [this](https://github.com/LBHackney-IT/repairs-api-dotnet/blob/90144321c41ca25b87858238a7852a87bc4c0e4c/RepairsApi/V2/UseCase/TenureInformationUseCase.cs#L49) referenced usage.